reviews/2022-cdk8s-annual.md

```diff
@@ -0,0 +1,101 @@
+## Background
+
+CDK For Kubernetes (cdk8s) is an open-source software development framework for defining Kubernetes 
+applications and reusable abstractions. Users leverage familiar programming languages and rich object-oriented APIs to 
+create standard Kubernetes manifests which can be applied to any Kubernetes cluster.
+
+CDK8s joined the CNCF on November 10, 2020.
+
+## Annual Review
+
+The following contains the requested content for the annual review, as described [here](https://github.com/cncf/toc/blob/main/process/sandbox-annual-review.md).
+
+### Include a link to your project’s devstats page.
+
+The full devstats page for cdk8s is available here: https://cdk8s.devstats.cncf.io/d/8/dashboards
+
+* In the past year, there were 7 [community contributions](https://cdk8s.devstats.cncf.io/d/56/company-commits-table?orgId=1&from=now-1y&to=now&var-repogroups=All&var-companies=Palo%20Alto%20Networks&var-companies=Cisco&var-companies=GridLine) 
+from 3 different users, in 3 different organizations (Palo Alto Networks, GridLine, Cisco). There is an additional [contribution](https://github.com/cdk8s-team/cdk8s-cli/pull/324) 
+that doesn’t show up on devstats, probably because the user isn’t associated with any company.
+
+### How many maintainers do you have, and which organizations are they from?
+
+The project currently has 4 maintainers, all from AWS:
```

Comment:
@iliapolo from https://github.com/cdk8s-team/cdk8s/blob/master/OWNERS.md it looks like there are 7 maintainers, all from AWS. Does OWNERS.md or this file need to be updated for accuracy?

Reply:
One maintainer (Ryan Parker) needs to be removed as he is no longer involved in the project. Everything else is accurate.

Reply:
Thanks for the PR! :)

Updated this file as well to reflect the correct state.